### PR TITLE
python: pyperf: Allow controlling PyPerf verbosity

### PR DESCRIPTION
**Pull request by Jongy**
Closes: https://github.com/Granulate/gprofiler/issues/516

### CODE REVIEW

File: gprofiler/profilers/python_ebpf.py [hunk 1]
```diff
@@ -125,13 +127,17 @@ def _kernel_stack_offset(self) -> int:
...
+    def _pyperf_base_command(self) -> List[str]:
+        cmd = [
+            resource_path(self.PYPERF_RESOURCE),
             "--fs-offset",
             str(self._kernel_fs_offset()),
             "--stack-offset",
             str(self._kernel_stack_offset()),
         ]
+        if self._verbose:
+            cmd.extend(["-v", "4"])
```
Comment (Jongy) [inline gprofiler/profilers/python_ebpf.py:139]: I'll document

File: gprofiler/profilers/python.py [hunk 2]
```diff
@@ -302,6 +302,7 @@ def _should_skip_process(self, process: Process) -> bool:
     "pyspy (always use py-spy), pyperf (always use PyPerf, and avoid py-spy even if it fails)"
     " or disabled (no runtime profilers for Python).",
     profiler_arguments=[
+        # TODO should be prefixed with --python-
```
[the remark this answers is not in the document]
Comment (Jongy) [reply gprofiler/profilers/python.py:305]: Nope but it's just a wrapper to `add_argument`. Anyway my point is, I don't want arguments to clobber the main namespace, so at some point I want to deprecare the arguments not prefixed with `--python-....`. Not for this PR

File: gprofiler/profilers/python_ebpf.py [hunk 3]
```diff
@@ -212,7 +218,7 @@ def _dump(self) -> Path:
             # also, makes sure its output pipe doesn't fill up.
             # using read1() which performs just a single read() call and doesn't read until EOF
             # (unlike Popen.communicate())
-            logger.debug(f"PyPerf output: {self.process.stderr.read1()}")  # type: ignore
+            logger.debug(f"PyPerf output: {self.process.stderr.read1(self._STDERR_READ_SIZE)}")  # type: ignore
```
[the remark this answers is not in the document]
Comment (Jongy) [reply gprofiler/profilers/python_ebpf.py:221]: Yup

**Review by d3dave — APPROVED** [review]

**Comment by Jongy** [comment]
Failures are not related.

**Pull request merged**